### PR TITLE
Clean common

### PR DESCRIPTION
**Pull request by SaraVieira**
This PR cleans a bunch of pages that still used the old Button and also some deletions:

- Deletes dynamic pages
- Deletes old Alert
- Deletes old deployment integration
- New Background and button on /s/github
- New Button on the import section of the new sandbox
- New button input in /cli/login
- New elements in /curator
- New elements in /dev/auth
- New Button in /s/github - Can we delete this page and redirect to create sandbox?
- New Button and elements in CodeSadbox
- New Button in the share modal
- Adds autoWidth prop to button
- Uses new design on Sandbox Error pages: s/dasdasd%20asd

Screenshot from codesadbox
CleanShot 2020-06-02 at 21 49 33@2x

Tip:

You can hide deleted files in the review
CleanShot 2020-06-02 at 21 59 43@2x

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/4295/commits/6012db22c33d8febf2a9b783a15197dfe43128e7 failed.

File: packages/components/src/components/Button/index.tsx [hunk 1]
```diff
@@ -108,10 +111,14 @@ export interface ButtonProps
   to?: string;
   as?: any;
   target?: any;
+  autoWidth?: boolean;
```
Comment (siddharthkp) [inline packages/components/src/components/Button/index.tsx:114]: can we call this prop `inline` which does the opposite of `block` in `Text` = 

display: inline-block
width: fit-content

Comment (SaraVieira) [reply packages/components/src/components/Button/index.tsx:114]: But it doesn't do that, it doesn't set to inline :(

Comment (siddharthkp) [reply packages/components/src/components/Button/index.tsx:114]: It should though? I imagine the intent here isn't just to make it width: auto but also for it to not take the whole block of space (display: block) and instead either take only as much space needed or space decided by it's parent (sounds like inline-block)

Comment (SaraVieira) [reply packages/components/src/components/Button/index.tsx:114]: Actually no, I literally just don't want it to span 100%

Most of these don't have any other thing set
CleanShot 2020-06-03 at 11 29 21

Comment (SaraVieira) [reply packages/components/src/components/Button/index.tsx:114]: In my head, those are two different things, I want them block but not 100%

Comment (SaraVieira) [reply packages/components/src/components/Button/index.tsx:114]: I can call it small?

Comment (SaraVieira) [reply packages/components/src/components/Button/index.tsx:114]: We decided to keep it for now

**Comment by DannyRuchtie** [comment]
Hi @SaraVieira 
Not sure this was already there. But I noticed the send button is not aligned:
Screenshot 2020-06-05 at 10 20 00

Also not sure this was part of your scope, but discovered this page still has old elements
`s/dasdasd%20asd`
Screenshot 2020-06-05 at 10 20 43

Hard to reproduce but the Back home button was not working for me (safari
Screenshot 2020-06-05 at 10 24 23

**Comment by SaraVieira** [comment]
@DannyRuchtie All the things mentioned are fixed

File: packages/app/src/app/pages/Sandbox/Editor/Header/Collaborators/AddCollaboratorForm.tsx [hunk 2]
```diff
@@ -86,7 +86,7 @@ export const AddCollaboratorForm = () => {
         />
       </motion.div>
 
-      <Button type="submit" css={css({ width: 'initial' })} paddingLeft={1}>
+      <Button type="submit" css={css({ width: 'initial' })} marginLeft={1}>
```
Comment (CompuIves) [inline packages/app/src/app/pages/Sandbox/Editor/Header/Collaborators/AddCollaboratorForm.tsx:89]: YES! Thank you!!

File: packages/app/src/app/pages/common/ErrorBoundary/CodeSadbox/CodeSadbox.tsx [hunk 3]
```diff
@@ -1,85 +1,94 @@
...
+                  <a href="/">
+                    <Button autoWidth variant="secondary">
+                      <ButtonIcon>
+                        <GoHome />
+                      </ButtonIcon>
+                      Back to Home
+                    </Button>
+                  </a>
+                )}
+                <a href={buildCrashReport({ error, trace })}>
```
Comment (CompuIves) [inline packages/app/src/app/pages/common/ErrorBoundary/CodeSadbox/CodeSadbox.tsx:74]: Can you bring the `errorCode` back here?

File: packages/app/src/app/pages/common/ErrorBoundary/CodeSadbox/buildCrashReport.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@ import VERSION from '@codesandbox/common/lib/version';
 interface IbuildCrashReport {
   error?: Error;
   trace?: string;
-  errorCode: string | null;
+  errorCode?: string | null;
```
Comment (CompuIves) [inline packages/app/src/app/pages/common/ErrorBoundary/CodeSadbox/buildCrashReport.ts:7]: Not needed as long as `errorCode` is given to the component again
```suggestion
  errorCode: string | null;
```

File: packages/app/src/app/pages/common/ErrorBoundary/CodeSadbox/CodeSadbox.tsx [hunk 5]
```diff
@@ -1,85 +1,94 @@
...
-  } = useOvermind();
-  const [errorCode, setErrorCode] = React.useState<string | null>(null);
+export const CodeSadbox: React.FC<IFallbackComponentProps> = withTheme(
+  ({ error, trace, theme }) => {
+    const {
+      actions: { codesadboxMounted },
+      state: { isLoggedIn },
+    } = useOvermind();
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const [_, setErrorCode] = React.useState<string | null>(null);
```
Comment (CompuIves) [inline packages/app/src/app/pages/common/ErrorBoundary/CodeSadbox/CodeSadbox.tsx:26]:
```suggestion
    const [errorCode, setErrorCode] = React.useState<string | null>(null);
```

File: standalone-packages/vscode-textmate/src/onigLibs.ts [hunk 6]
```diff
@@ -1,7 +1,8 @@
 /*---------------------------------------------------------
```
Comment (CompuIves) [inline standalone-packages/vscode-textmate/src/onigLibs.ts:1]: Could you revert this file to master? This one needs to stay in sync with VSCode.

File: standalone-packages/codesandbox-browserfs/src/backend/CodeSandboxFS.ts [hunk 7]
```diff
@@ -172,9 +172,9 @@ export default class CodeSandboxFS extends SynchronousFileSystem
```
Comment (CompuIves) [inline standalone-packages/codesandbox-browserfs/src/backend/CodeSandboxFS.ts:172]: Can your revert this file?

File: packages/app/src/app/pages/common/ErrorBoundary/CodeSadbox/CodeSadbox.tsx (hunk 3 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/common/ErrorBoundary/CodeSadbox/CodeSadbox.tsx]: I have no idea why I deleted it tbh

File: standalone-packages/vscode-textmate/src/onigLibs.ts (hunk 6 above)
Comment (SaraVieira) [reply standalone-packages/vscode-textmate/src/onigLibs.ts]: Done, no idea what happened

File: standalone-packages/codesandbox-browserfs/src/backend/CodeSandboxFS.ts (hunk 7 above)
Comment (SaraVieira) [reply standalone-packages/codesandbox-browserfs/src/backend/CodeSandboxFS.ts]: Done, no idea what happened

**Pull request merged**